### PR TITLE
Funqy Google Cloud Functions codestart

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/singleton-examples/funqy-google-cloud-functions-example/base/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Funqy Google Cloud Functions Integration examples
+
+Examples of Background Functions with Funqy for Quarkus.
+
+Guide: https://quarkus.io/guides/funqy-gcp-functions
+
+Two examples have been generated under `src/main/java/org/acme/fuqygooglecloudfunctions`, you must remove them before deploying to
```
Comment (ia3andy) [inline devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/singleton-examples/funqy-google-cloud-functions-example/base/README.md:7]:
```suggestion
Two examples have been generated under `src/main/java/org/acme/funqygooglecloudfunctions`, you must remove them before deploying to 
```

**Review by ia3andy — CHANGES_REQUESTED** [review]
Cool!

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/singleton-examples/funqy-google-cloud-functions-example/base/README.md (hunk 1 above)
Comment (loicmathieu) [reply devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/singleton-examples/funqy-google-cloud-functions-example/base/README.md]: Nice catch, fixed!

**Comment by gsmet** [comment]
@ia3andy could you have another look?

**Comment by ia3andy** [comment]
I added a new `metadata` section in the codestart.yml, and added a template for the readme. I will push a commit today to update this PR with it..

**Comment by loicmathieu** [comment]
> I added a new metadata section in the codestart.yml, and added a template for the readme. I will push a commit today to update this PR with it..

@ia3andy do you have time to do it today . 1.12 will be cut off tomorrow I would like to have this one included inside it

**Comment by ia3andy** [comment]
Ok! I will :)

**Comment by ia3andy** [comment]
@loicmathieu could you rebase the PR from master please?

**Comment by ia3andy** [comment]
I created a new PR from yours https://github.com/quarkusio/quarkus/pull/14942

**Pull request closed without merging**